### DISCUSSION TITLE
Send notes at the same time

### DISCUSSION THREAD

**Discussion by JanvyHung** [mido/mido#421] [Q&A] [answered]
if I need to add several notes at same time ,how can i do?

**Comment by jholloway7** [comment, marked as answer]
You send the 'note_on' events in rapid succession. This isn't a 'mido' limitation, per se, as there is no way in MIDI to send them 'grouped' together. 

That might seem strange, but consider that even when you're playing an instrument live, you don't precisely hit the notes at the exact same time even if you perceive that you do.  There are usually imperceptible nano-/micro-second delays as your fingers strike the keys, strings whatever.
